_ap/18/index.adoc

```diff
@@ -3,76 +3,49 @@ num: 18
 title: "OpenAPI clients code generation - Kiota"
 status: "Draft"
 authors:
-- "andreaTP"
+- "Andrea Peruffo"
 tags: []
 ---
 
-// Top style tips:
-// * Use one sentence per line
-// * No unexpanded acronyms
-// * No undefined jargon
-
-// You don't have to use the following sections, but they provide a 
-// useful structure for writing a clear document.
-
 ## Intent
-// Summarize in a single sentence what the pattern tries to achieve.
-//
-// For example:
-//
-// Define a policy for exposing metrics to customers in a consistent way.
+
+Define a consistent technology to be used for client code generation from OpenAPI specifications.
 
 ## Motivation
 
-// In a few paragraphs describe the motivating factors for this pattern
-//
-// For example:
-// 
-// Prometheus scrape endpoints are the defacto standard for exposing metric information for collection.
-// However this format and the associated conventions are not sufficient in themselves to ensure
-// that metrics are exposed in a consistent way. Problems include:
-// * Insufficient guidance on metric and label naming to provide consistency and 
-//   establishing the principal of hiding implementation details.
-// * Treatment of this as a first-class API, with established mechanisms for API evolution
-// * etc.
+* Have a uniform and good quality SDK layer to be used for APIs access
+* Be able to maintain the generated code and the necessary libraries
+* Ability to fix bugs and eventually improve the generator codebase
 
 ## Applicability
 
-// Call out when this pattern might apply (and when it should not apply, if relevant)
-// For example:
-//
-// This pattern applies to any service which exposes metrics as Prometheus-scrape endpoints.
-// It does not apply to services which provide access to metrics in other formats, or via
-// means other than an API (e.g. metric visualization via a system such as Grafana).
+This pattern applies to any service using a client/SDK to access REST API endpoints.
```

Comment:
```suggestion
This pattern applies to any service providing a client/SDK to access REST API endpoints.
```

Reply:
this is actually not correct, Kiota is going to be used on the "receiver side", and possibly even without the need of curated SDKs

Reply:
So the intent is not to offer SDKs (generated). But to use Kiota direcly in the CLI and UI build system?

Reply:
integrated with `using or providing`